src/action/base/create_user.rs

```diff
@@ -178,31 +178,57 @@ impl Action for CreateUser {
                 .await?;
             },
             _ => {
-                execute_command(
-                    Command::new("useradd")
-                        .process_group(0)
-                        .args([
-                            "--home-dir",
-                            "/var/empty",
-                            "--comment",
-                            &format!("\"Nix build user\""),
```

Comment:
This ended up putting `"Nix build user"` (including the quotes) in `/etc/passwd`, but e.g. NixOS and the official install scripts have it without the quotes.

(Also, the official scripts include the user's number... We might want to do the same.)

Reply:
We should, yes!